UnityMirror/UnityServer/Assets/PlayFabEditorExtensions.meta

```diff
@@ -1,9 +0,0 @@
-fileFormatVersion: 2
```

Comment:
This file was not needed to be commited in the first place or is it redundant now?